frontend/src/ui/components/Header/Header.tsx

```diff
@@ -1,74 +1,51 @@
 import * as React from 'react';
-import { Navbar, Nav, Container } from 'react-bootstrap';
-import { ROUTES, CONTACT_US_LINK } from 'global/constants';
-import { useLocation, useHistory } from 'react-router-dom';
+import { Container, Nav, Navbar } from 'react-bootstrap';
+import { CONTACT_US_LINK, ROUTES } from 'global/constants';
+import { Route, useHistory } from 'react-router-dom';
 import logo from 'assets/icons.svg';
 import './styles.scss';
 
 export const Header: React.FC = () => {
-  const currentLocation = useLocation().pathname;
   const history = useHistory();
 
-  // Workaround to check authentication on header instead of connecting
-  // component to redux
-  const authenticated = currentLocation.includes(ROUTES.Console.HOME);
-
-  if (authenticated) {
-    return (
-      <Container>
-        <Navbar bg="transparent" expand="lg" variant="dark">
-          <Navbar.Brand className="logo-container mx-auto">
-            <svg className="site-logo">
-              <use xlinkHref={`${logo}#opencraft_logo`} />
-            </svg>
-          </Navbar.Brand>
-          <Navbar.Toggle aria-controls="basic-navbar-nav" />
-          <Navbar.Collapse id="basic-navbar-nav">
-            <Nav className="ml-auto">
-              <Nav.Link>Customize</Nav.Link>
-              <Nav.Link onClick={() => window.open(CONTACT_US_LINK, '_blank')}>
-                Support Request
-              </Nav.Link>
-              <Nav.Link disabled>Status & Notifications</Nav.Link>
-              <Nav.Link
-                onClick={() => {
-                  history.push(ROUTES.Auth.LOGOUT);
-                }}
-              >
-                Log out
-              </Nav.Link>
-            </Nav>
-          </Navbar.Collapse>
-        </Navbar>
-      </Container>
-    );
-  }
+  const navigateTo = (route:string) => () => history.push(route);
 
   // Only show login link when on registration page and only show registration
   // link when on login page
   return (
     <Container>
       <Navbar bg="transparent" expand="md" variant="dark">
         <Navbar.Brand className="logo-container mx-auto">
-          <svg className="site-logo">
-            <use xlinkHref={`${logo}#opencraft_logo`} />
-          </svg>
+          <Nav.Link onClick={navigateTo(ROUTES.Console.HOME)}>
+            <svg className="site-logo">
+              <use xlinkHref={`${logo}#opencraft_logo`}/>
+            </svg>
+          </Nav.Link>
```

Comment:
The logo is now linked to the home page.

Reply:
This was the main reason for doing that! Otherwise, redirecting would either go to the login page when logged out, or the registration page when logged in.